allofplos/gdrive.py

```diff
@@ -0,0 +1,160 @@
+import datetime
+import os
+import tarfile
+from zipfile import ZipFile, BadZipFile
+
+import requests
+from tqdm import tqdm
+
+from . import get_corpus_dir
+
+# Variables needed
+zip_id = '0B_JDnoghFeEKLTlJT09IckMwOFk'
+metadata_id = '0B_JDnoghFeEKQUhKWXBOVy1aTlU'
```

Comment:
what's the difference between these ids?

Reply:
these point at 2 different files on google drive. the zip_id is the actual zip, and the metadata_id is the text file with zip file size and date info